dali/operators/reader/coco_reader_op.cc

```diff
@@ -133,6 +133,12 @@ instance of an object is lower than this value, the object will be ignored.)code
   .AddOptionalArg("image_ids",
       R"code(If set to True, the image IDs will be produced in an extra output.)code",
       false)
+  .AddOptionalArg("avoid_class_remapping",
+      R"code(If set to False, classes' ID values are mapped to consecutive values in range 1-number of classes
+disregarding exact values from the manifest (0 is reserved for a special background class).
+
+Otherwise classes ID values are returned directly as they are defined in the configuration file.)code",
```

Comment:
I think that these paragraphs should be swapped: first, say what happens when the user specifies a non-default value and then (otherwise) state what happens when it's left with a default value.

Reply:
Done